addons/jest/src/index.js

```diff
@@ -1,7 +1,8 @@
 import addons from '@storybook/addons';
+import deprecate from 'util-deprecate';
 
 const findTestResults = (testFiles, jestTestResults, jestTestFilesExt) =>
-  testFiles.map(name => {
+  [].concat(testFiles).map(name => {
```

Comment:
why ? `map()` is immutable no ?

Reply:
`[].concat()` is a way to "ensure array" -- `[].concat('a') == [].concat(['a'])`

Reply:
So it lets the user set `{ jest: 'filename' }` or `{ jest: ['filename', 'other-filename'] }` which is natural.

Reply:
`Array.from(testFiles)` is a more explicit way to do the same